observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,172 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+To configure forwarding logs to a LokiStack instance, you can deploy and configure the OpenTelemetry Collector. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {loki-op} is installed. See _Additional resources_ on how to install LokiStack.
+* A supported LokiStack instance is deployed on the cluster.
+
+TODO add dev-preview note and that the `lokipush` exporter will be removed in future release in favour of OTLPHTTP.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.Example ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+----
+
+. Create a cluster role that gives collector service account permissions to push logs to the LokiStack application tenant. The cluster role has also permissions for `k8sattributes` processor.
```

Comment:
@pavolloffay, the second sentence " The cluster role has also permissions for `k8sattributes` processor," but I don't see this in the following example. Could you please have a second look at the example?

Reply:
The second sentence can be removed

Reply:
Done. Thank you.